### ISSUE TITLE
About the Transition1x given in this repository

### ISSUE THREAD

**Issue by allblueJT** [chenruduan/OAReactDiff#5]
Hello, there seems to be some additional properties in the Transitions1x given in this repo (train_addprop.pkl) compared to the origin, e.g. SMILES of the reactant/product. Could you tell me how to get such SMILES strings?  And some of the SMILES strings caused a failure while using rdkit.Chem.MolFromSmiles(), which means an illegal SMILES string was provided.

**Comment by zEuler1027** [comment]
SMILES comes with the trainsition1x dataset and I think most likely it comes from QM9.

**Comment by Songze1019** [comment]
SMILES comes with the trainsition1x dataset and I think most likely it comes from GDB7 dataset.

**Comment by allblueJT** [comment]
> SMILES comes with the trainsition1x dataset and I think most likely it comes from GDB7 dataset.

Only rxns are provided in the Transition1x without correspondence between rxn and SMILES. I wonder how to get this correspondence..

Moreover, not all SMILES strings in the train_addprop.pkl are in the GDB7, e.g. O[C]1=NOC=[N]=1

**Comment by Zhanghahah** [comment]
> > SMILES comes with the trainsition1x dataset and I think most likely it comes from GDB7 dataset.
> 
> Only rxns are provided in the Transition1x without correspondence between rxn and SMILES. I wonder how to get this correspondence..
> 
> Moreover, not all SMILES strings in the train_addprop.pkl are in the GDB7, e.g. O[C]1=NOC=[N]=1

Hello! Have you found the mapping relations between 'rxn' in Transition1x and GDB7? Thanks!

**Comment by chenruduan** [comment]
I do not think this is possible. Better ask Matthias, the main contributor of transition1x dataset (https://www.nature.com/articles/s41597-022-01870-w). I will close now since this issue is not that relavant to OA-ReactDiff.
